opm/common/OpmLog/LogBackend.cpp

```diff
@@ -27,18 +27,37 @@ namespace Opm {
     {
     }
 
+    LogBackend::~LogBackend()
+    {
+    }
 
-    bool LogBackend::includeMessage(int64_t messageFlag) {
-        if (((messageFlag & m_mask) == messageFlag) &&
-            (messageFlag > 0))
-            return true;
-        else
-            return false;
+    void LogBackend::configureDecoration(bool use_prefix, bool use_color_coding)
```

Comment:
With the risk of going overboard; we could implement a `std::unorderd_map<int64_t , std::string>`with per message type prefix (and color) configuration.

```C++
void LogBackend::configurePrefix( int64_t messageType , const std::string& prefix) {

}

void LogBackend::configureColor(int64_t messageType, const ColorType& color)  {

}
```